packages/react-ui-core/src/List/List.js

```diff
@@ -18,7 +18,13 @@ export default class List extends PureComponent {
       PropTypes.func,
       PropTypes.object,
     ]),
-    items: PropTypes.array,
+    items: PropTypes.arrayOf(
+      PropTypes.oneOfType([
+        PropTypes.node,
+        PropTypes.func,
```

Comment:
is there existing test case for when items are functions?

Reply:
Yes, that could be `() => (<div>foo</foo>)`.

Reply:
Good catch @lfender6445 ! Test added - and bug fixed!